pkger/service.go

```diff
@@ -889,32 +889,32 @@ func (s *Service) dryRunVariables(ctx context.Context, orgID influxdb.ID, pkg *P
 	mExistingLabels := make(map[string]DiffVariable)
 	variables := pkg.variables()
 
-VarLoop:
+	existingVars, _ := s.varSVC.FindVariables(ctx, influxdb.VariableFilter{
+		OrganizationID: &orgID,
+		// TODO: would be ideal to extend find variables to allow for a name matcher
```

Comment:
Is it worth doing this now? Will this not work correctly for users with 100+ variables?

Reply:
we only make this call once per run. It could be done now, but definitely a nice to have.

Reply:
the VarSVC doesn't support a find by name atm 😢

Reply:
@jademcgough I pushed up the pagination changes to make sure we get all variables instead of only the first 100 👍